### PR TITLE
Fix: Position of  parameter  is always changing on addon's details page.

### LINKED ISSUE #3344: position of  parameter  is always changing on addon's details page.
**Describe the bug**
The

**To Reproduce**
1. install velaux
2. goto "Addons"
3. click one addon such as "terraform-aws" to view its details
4. api "http://velaux/api/v1/addons/terraform-aws" returns order of configuration parameters in 'uiSchema.(configuration parameters).Sort'
5. if the addon is not installed yet, or the configuration parameter is not specified sort value in "UI Schema ",  "sort" is set to be 100 by default
6. on addon detail page, the placement of parameter with default sort value is randomly. That means the position of  parameter  is always changing when you view addon's details page. 

**Expected behavior**
1.should not be randomly

**Screenshots**

**KubeVela Version**

**Cluster information**

**Additional context**

### PR DESCRIPTION
**Pull request by StevenLeiZhang**
Signed-off-by: StevenLeiZhang <zhangleiic@163.com>

### Description of your changes

Fixes #3344 

Before:
The default value of "sort" is 100

After:
Set fixed value for every parameter

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: pkg/apiserver/rest/usecase/definition.go [hunk 1]
```diff
@@ -318,6 +318,12 @@ func renderDefaultUISchema(apiSchema *openapi3.Schema) []*utils.UIParameter {
 			params = append(params, param)
 		}
 	}
+	sort.Slice(params, func(i, j int) bool {
```
Comment (barnettZQG) [inline pkg/apiserver/rest/usecase/definition.go:321]: Suggested sorting conditions:
1. Check `validate.required`.  It is True, the sort number will be lower.
2. Check `subParameters`. The more subparameters, the larger the sort number.
3. The sort number starts at 100.

**Review by wonderflow — APPROVED** [review]
great work!

File: pkg/apiserver/rest/usecase/definition.go (hunk 1 above)
Comment (StevenLeiZhang) [reply pkg/apiserver/rest/usecase/definition.go:332]: fixed.

**Comment by StevenLeiZhang** [comment]
> @StevenLeiZhang Please fix the CI error.

Fixed

**Review by barnettZQG — APPROVED** [review]
LGTM

**Pull request merged**